docs/node-observ-lib/annotations.libsonnet

```diff
@@ -0,0 +1,69 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+{
+  new(this):
+    {
+      reboot:
+        commonlib.annotations.reboot.new(
+          title='Reboot',
+          target=this.grafana.targets.reboot,
+          instanceLabels=std.join(',', this.config.instanceLabels),
+        )
+        + commonlib.annotations.base.withTagKeys(std.join(',', this.config.groupLabels + this.config.instanceLabels)),
+      memoryOOM:
+        commonlib.annotations.base.new(
+          'OOMkill',
+          this.grafana.targets.memoryOOMkiller
+        )
+        + commonlib.annotations.base.withTagKeys(std.join(',', this.config.groupLabels + this.config.instanceLabels))
+        + commonlib.annotations.base.withTextFormat('')
+          {
+          hide: true,
+          iconColor: 'light-purple',
+        },
```

Comment:
Couldn't you use `commonlib.annotations.fatal.new` instead? These values are implied there 🤔

Reply:
Technically, it is not fatal event. So maybe it is best to change color instead :)